test/AffInterface.purs

```diff
@@ -53,6 +41,8 @@ addr1 :: OgmiosAddress
 addr1 =
   "addr1qyc0kwu98x23ufhsxjgs5k3h7gktn8v5682qna5amwh2juguztcrc8hjay66es67ctn0jmr9plfmlw37je2s2px4xdssgvxerq"
 
+type TestPlanM a = MoteT (Const Void) (QueryM Unit) Aff a
```

Comment:
Another thing that I'd prefer if possible: can we test through the `Contract` interface instead? I don't see a reason to use `QueryM` directly (I know these tests used to use `runQueryM`, but I think they actually predate our `Contract` interface)

Reply:
I've had a look into this, for the most part it'd be straight forward, but the submit tx failure case would be a little more verbose due to the input construction. Also, I'm not sure I agree anyway - the point of these tests are to test the websocket interface (maybe we should rename the module), `QueryM` being a fairly light wrapper around them.

Reply:
Hm, perhaps I should move

```purescript
  group "Address loop" do
    test "Ogmios Address to Address & back Testnet"
      $ testFromOgmiosAddress testnet_addr1
    test "Ogmios Address to Address & back non-Testnet"
      $ testFromOgmiosAddress addr1
```

elsewhere too, these don't even use `QueryM`